### ISSUE TITLE
Changing default session access level for worlds opened by URL/resrec

### ISSUE THREAD

**Issue by BitCrack** [Yellow-Dog-Man/Resonite-Issues#1495]
### Is your feature request related to a problem? Please describe.

This is an issue I came across while reviewing this year's MMC24 entries. Opening a world by URL e.g. `resrec:///U-1PrDkbLC4m0/R-b0639933-bf48-4686-8197-67853fe78e0a` opens a new session with the `Anyone` access level. I prefer going through entries alone in peace and every time I open a world I have to change the access level of the session which gets kind of tedious and I often forget to actually do that.

### Describe the solution you'd like

Given the settings re-work is in progress, it'd be nice to have a "Default session access level" setting or anything similar that could possibly affect:
- The initial access level for any sessions created by the user when pasting a world URL
- The initial access level for any sessions created by the user when clicking "Start Session" while viewing world's sessions
- The default selected access level when opening the "Start Custom Session" prompt

### Describe alternatives you've considered

The only other solution to this problem is making a mod, which I haven't tried making myself.

**Comment by Frooxius** [comment]
We could add a setting for this once setting rework is done.

In the meanwhile, I'd recommend using the "Start Custom Session" and selecting level there.

Another current workaround is to also set yourself as Invisible - this will make all sessions default to private by default.

**Comment by XDelta** [comment]
Tried just now, setting yourself to invisible seems to set the world as Private, With all other status setting to anyone. This appears to remain consistent with all manors of opening worlds. If you are invisible, worlds you open will set to Private unless you then change that in the session settings or start a custom session.
